packages/bech32m/src/format/index.ts

```diff
@@ -31,12 +31,6 @@ export default {
     byteLength: ByteLength.penumbra,
     innerName: Inner.penumbra,
   },
-  penumbracompat1: {
```

Comment:
It might be worth keeping it around to render historical txs etc.

Reply:
However, it would be good to add the transparent address prefix so that the bech32 library has the ability to encode an `Address` etc.

Here's the code in core: https://github.com/penumbra-zone/penumbra/blob/020986747245270a00d039306458be9e7f1f33f5/crates/core/keys/src/address.rs#L232

Reply:
that's a good suggestion, would enable rendering a t-addr in prax 👍

Reply:
added back the `compat` bech32 encoding and added the [transparent](https://github.com/penumbra-zone/web/pull/1950/commits/486e9a8d98ec87964c72695fbe9233e6b6fa07a7) address prefix to the bech32 library.